src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1873,16 +1887,20 @@ private void printDevModeMessages(boolean inputUnavailable, boolean startup) {
             hotTests = true;
         }
         if (startup) {
+            info(formatAttentionMessage(""));
+            info(formatAttentionTitle("Liberty server port information:"));
             if (httpPort != null) {
                 if (container) {
-                    info(formatAttentionMessage("Liberty server HTTP port mapped to Docker host port: " + httpPort));
+                    info(formatAttentionMessage("Internal container HTTP port: " + containerHttpPort));
+                    info(formatAttentionMessage("Mapped Docker host HTTP port: " + httpPort));
```

Comment:
If the internal and mapped ports are the same shouldn't we only print it once? Maybe we could use a message that explains that or maybe we update the docs to explain it?

Reply:
This will be addressed in https://github.com/OpenLiberty/ci.maven/issues/1000